web/src/main/java/se/inera/intyg/webcert/web/web/controller/api/IntygApiController.java

```diff
@@ -272,7 +277,12 @@ public Response getEventsForCertificate(@PathParam("certificateId") String certi
             LOG.debug("No events for certificate with id {}", certificateId);
         }
 
-        return Response.ok(eventList).build();
+        List<CertificateEventDTO> eventDTOS = eventList
```

Comment:
If you are going to change any of my other comments i suggest renaming the variable to eventDTOList.